lib/xlsxtream/xml.rb

```diff
@@ -36,8 +39,19 @@ def escape_attr(string)
         string.gsub(UNSAFE_ATTR_CHARS, XML_ESCAPES)
       end
 
+      # Ensure that we escape the first underscore character in plaintext strings that match the format for Excel escape sequences.
+      # This ensures that these strings are displayed as plaintext and not incorrectly parsed as escape sequences by Excel
+      # Per Microsoft Open Specifications for Excel:
```

Comment:
I just approved, but I think we should link to the spec doc in this comment. Makes it clear what 'spec' we're referencing, and will help future devs of this gem once this is merged to the main gem

Reply:
Updated: https://github.com/Invoca/xlsxtream/pull/1/commits/2e39341c4b8f8a60075ff185aa318895ca806ad4